platformio_orig.ini

```diff
@@ -126,6 +127,11 @@ upload_protocol = custom
 
 [env:usb]
 upload_protocol = esptool
+upload_speed = 921600
```

Comment:
This setting is global, for all type boards. Thus, we can't have this here, because some older boards do not support high upload speed. Please remove.